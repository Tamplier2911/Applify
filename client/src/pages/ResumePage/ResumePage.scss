/*
.resume-page {
  display: grid;

  @media only screen and (max-width: 768px) {
  }

  @media only screen and (max-width: 540px) {
  }

  @media only screen and (max-width: 425px) {
  }
}

*/
