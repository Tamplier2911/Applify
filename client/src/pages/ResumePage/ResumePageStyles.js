import styled from "styled-components";

export const ResumePageContainer = styled.section`
  display: grid;

  @media only screen and (max-width: 768px) {
  }

  @media only screen and (max-width: 540px) {
  }

  @media only screen and (max-width: 425px) {
  }
`;
