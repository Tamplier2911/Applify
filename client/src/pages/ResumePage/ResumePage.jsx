// import "./ResumePage.scss";
import React from "react";
import PropTypes from "prop-types";

// components
import Resume from "../../components/Resume/Resume";

// JS Rendering CSS
import { ResumePageContainer } from "./ResumePageStyles";

const ResumePage = () => {
  return (
    <ResumePageContainer data-test="resume-page">
      <Resume data-test="resume-page-child" />
    </ResumePageContainer>
  );
};

ResumePage.propTypes = {
  success: PropTypes.bool,
  // success: PropTypes.bool.isRequired,
};

export default ResumePage;
