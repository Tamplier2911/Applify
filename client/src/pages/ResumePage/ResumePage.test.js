import React from "react";
// import { Provider } from "react-redux";

// component
import ResumePage from "./ResumePage";

// utils
import {
  findByTestAttr,
  setShallow,
  // setRender,
  // setMount,
  checkProps,
  // createStore,
} from "../../tests/testUtils";

describe("<ResumePage />", () => {
  it("renders without error", () => {
    const wrapper = setShallow(<ResumePage />);
    const resumePage = findByTestAttr(wrapper, "resume-page");
    expect(resumePage.length).toBe(1);
  });

  it("renders children", () => {
    const wrapper = setShallow(<ResumePage />);
    const resumePageChild = findByTestAttr(wrapper, "resume-page-child");
    expect(resumePageChild.length).toBe(1);
  });

  it("recieves correct props", () => {
    const expectedProps = { success: true };
    const err = checkProps(ResumePage, expectedProps, "success");
    expect(err).toBeUndefined();
  });

  it("matches snapshot", () => {
    const wrapper = setShallow(<ResumePage />);
    const resumePage = findByTestAttr(wrapper, "resume-page");
    expect(resumePage).toMatchSnapshot();
  });
});
