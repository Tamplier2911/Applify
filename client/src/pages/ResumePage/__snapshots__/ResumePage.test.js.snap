// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResumePage /> matches snapshot 1`] = `ShallowWrapper {}`;
